finality-aleph/src/network/gossip/service.rs

```diff
@@ -5,14 +5,16 @@ use std::{
     hash::Hash,
 };
 
-use futures::{channel::mpsc, StreamExt};
+use futures::{
+    channel::{mpsc, mpsc::TrySendError},
+    StreamExt,
+};
 use log::{debug, error, info, trace, warn};
 use network_clique::SpawnHandleT;
 use rand::{seq::IteratorRandom, thread_rng};
-use sc_utils::mpsc::{tracing_unbounded, TracingUnboundedReceiver, TracingUnboundedSender};
 use tokio::time;
 
-const QUEUE_SIZE_WARNING: usize = 1_000;
+const MAX_QUEUE_SIZE: usize = 1_000;
```

Comment:
We've recently seen the limit of 1000 being hit - just a question, if we're gonna drop the messages, do we want to increase this limit?

Reply:
If anything probably decrease – neither block sync nor authentications should be sending that many messages to a single node ever.

Reply:
I'd keep it as it is and let us make relevant benchmarks to determine optimal value there later, in the task related to bounding memory usage in the whole aleph-node - probably this value should be editable in some config anyway.